sql_generators/table_partition_expirations/templates/table_partition_expirations.sql

```diff
@@ -55,6 +55,14 @@ current_partitions AS (
     {% endif %}
   {% endfor %}
 ),
+partition_expirations AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.region-us.INFORMATION_SCHEMA.TABLE_OPTIONS`
```

Comment:
Is it always in region-us or should this be {{ dataset_id }} also?

Reply:
This is outside of the for loop so `region-us` works here because we can just get options for every table in the project and left join them which is faster.  `{{ dataset_id }}` is used for `INFORMATION_SCHEMA.PARTITIONS` since it only supports that